src/test/java/uk/gov/companieshouse/mapper/AppealMapperTest.java

```diff
@@ -22,7 +22,7 @@
 import uk.gov.companieshouse.model.PenaltyIdentifier;
 import uk.gov.companieshouse.model.Reason;
 
-@ExtendWith(SpringExtension.class)
+@ExtendWith(MockitoExtension.class)
 public class AppealMapperTest {
```

Comment:
Remove **public** from class declaration. Also, we use the **@DisplayName** annotation on test methods to give a more verbose way of describing a test. Can this annotation be added to test methods in the class please.

Reply:
Done in new commit